nat-lab/tests/test_teliod.py

```diff
@@ -66,7 +67,99 @@ async def test_teliod() -> None:
         assert not teliod_process.is_executing()
         assert not await is_teliod_running(connection)
 
-        # Run the hello-world command again - this time it should fail
+        # Run the get-status command again - this time it should fail
         with pytest.raises(ProcessExecError) as err:
             await connection.create_process(TELIOD_STATUS_PARAMS).execute()
         assert err.value.stderr == "Error: DaemonIsNotRunning"
+
+
+async def test_teliod_meshnet_toggle() -> None:
```

Comment:
this test is testing the behaviour of the status reporting and not truly if the meshnet is working

Reply:
That is correct, since we don't have an API token and any actual peers for the account, I am not exactly sure how we could test it. What do you propose?

Reply:
API is mocked for every nat-lab tests that use telio client (`tcli`), check: [API](https://github.com/NordSecurity/libtelio/blob/main/nat-lab/tests/mesh_api.py#L208) and [clients](https://github.com/NordSecurity/libtelio/blob/main/nat-lab/tests/helpers.py#L216) setup.

I propose a real scenario, launching `teliod` and testing the meshnet connection between two or more peers.

Reply:
Removed these tests from this PR